workflows/differentialabundance.nf

```diff
@@ -531,6 +575,14 @@ workflow DIFFERENTIALABUNDANCE {
             )
     }
 
+    if (params.gprofiler2_run){
+        ch_report_input_files = ch_report_input_files
+            .combine(GOST.out.plot_html.map{it[1]}.flatMap().toList())
+            .combine(GOST.out.all_enrich.map{it[1]}.flatMap().toList())
+            .combine(GOST.out.sub_enrich.map{it[1]}.flatMap().toList())
+        GOST.out.plot_html
```

Comment:
Should there be a combine here?

Reply:
Oops, that was a copy-paste-error, the channel is already combined in that block